config/environments/development.rb

```diff
@@ -50,7 +50,7 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  config.i18n.available_locales = [:en, :es, :de, :fr, :et, :ca, :ru, :ja, :hi, 'zh-cn', 'zh-tw', 'pt', :nl, 'tr', 'pt-br']
+  config.i18n.available_locales = [:en, :es, :de, :fr, :et, :ca, :ru, :ja, :hi, 'zh-cn', 'zh-tw', 'pt', :nl, 'tr', 'pt-br', :ar]
   config.i18n.default_locale = :en
```

Comment:
Because Helpy overwrites the available_locales and controls this by a web UI, to get this working you need to add the locale file, mention it here and then "enable" it in your dev copy of helpy in the settings area.